decompiler/config/all-types.gc

```diff
@@ -6906,7 +6903,7 @@
 (define-extern *ocean-trans-up-left-table* (pointer float))
 (define-extern *ocean-trans-corner-table* (pointer float))
 (define-extern *ocean-trans-strip-array* (pointer float))
```

Comment:
maybe this is type `ocean-trans-strip-array`?

Reply:
that would definitely make sense, but it doesn't decompile
```mips
    lw t0, *ocean-trans-strip-array*(s7);; [161] (set! t0-5 *ocean-trans-strip-array*)
                              ;; [] -> [t0: ocean-trans-strip-array ]
    daddu a3, a3, t0          ;; [162] (set! a3-17 (+ a3-16 t0-5))
                              ;; [a3: <integer 0 + (160 x int)> t0: ocean-trans-strip-array ] -> [a3: ocean-trans-strip ]
    dsll32 a3, a3, 1          ;; [163] (set! a3-18 (sll a3-17 33)) [a3: ocean-trans-strip ] -> [a3: <uninitialized> ]
    dsrl a3, a3, 1            ;; [164] (set! a3-19 (srl a3-18 1)) [a3: <uninitialized> ] -> [a3: <uninitialized> ]
    or a2, a2, a3             ;; [165] (set! a2-18 (logior a2-17 a3-19))
                              ;; [a2: <uninitialized> a3: <uninitialized> ] -> [a2: <uninitialized> ]
```
> Function ocean-trans-add-upload-strip failed type prop at op 163: Cannot get_type_int2: (sll a3 33), args ocean-trans-strip and <integer 33>

If I make it a pointer to a ocean-trans-strip-array it works but it still looks like a badly formed array access.

Reply:
ok, let's leave it at `(pointer float)` for now then.